plugin/src/main/java/org/opensearch/ml/action/models/DeleteModelTransportAction.java

```diff
@@ -234,6 +262,106 @@ public void onFailure(Exception e) {
         });
     }
 
+    private void checkAgentBeforeDeleteModel(String modelId, ActionListener<Boolean> actionListener) {
```

Comment:
The current method order doesn't align with the call hierarchy, making it harder to follow.

Reply:
Sure. Already reorder the functions.